### ISSUE TITLE
Map App opens up, when selecting world map in suntimes

### ISSUE THREAD

**Issue by hacksenwerk** [forrestguice/SuntimesWidget#573]
Samsung Galaxy S5 - SM-G900F
LineageOS 18.1 
Suntimes 0.14.1 (a3ff0b18b) from f-droid.org

When you open the app, there's a worldmap icon in the upper right. 
When I select it, my map app "Organic Maps" opens up. 

When I select the three-dot-menu and inside that select the Worldmap, it opens the suntimes world map, as it should.

**Comment by forrestguice** [comment]
Thanks for the report - this is how its designed. The map icon in the action bar (always displayed) opens the currently configured location in some other app. Meanwhile the world map dialog is buried in the overflow menu.

They share an icon which isn't ideal - maybe we can pick a better icon for one or both of these actions.

**Comment by hacksenwerk** [comment]
Oh I see... I didn't know that. 

Thank you for your reply.

**Comment by forrestguice** [comment]
Yeah, its not a "bug", but these kinds of reports are still very helpful for improving the UI. I appreciate it.
We need to differentiate those two somehow. I think that action bar icon could be made configurable too.
